platform/src/main/java/org/stellar/anchor/platform/config/PropertySep10Config.java

```diff
@@ -15,4 +15,5 @@ public class PropertySep10Config implements Sep10Config {
   private List<String> clientAttributionDenyList;
   private List<String> clientAttributionAllowList;
   private List<String> omnibusAccountList;
+  private boolean requireKnownOmnibusAccount = true;
```

Comment:
idea behind this is `secure by default`. also, i believe omnibusAccountList is empty on our deployments so this shouldnt break anything. please let me know if you feel differently.

Reply:
Yea, I would communicate with Bitso. Also they use helm chart so we should make sure the pipeline is updated with helm before we land this in main.

Reply:
AFAIU, `requireKnownOmnibusAccount` would not make it secure :-). It is an option to enforce `omnibusAccountList`. This will make non-custodial wallet to fail. 

IMO, if we want to make Anchor Platform to work with non-custodial wallet, we should make the default to `false`. But if we want to make Anchor Platform to work with omnibus wallet only, then we set default to 'true'.

Reply:
anchor-config-defaults.yaml has the value set to false and that will work as the default value now. removed the java object default value